tests/parse_metadata.py

```diff
@@ -240,6 +246,16 @@ def read_metadata(distribution: str) -> StubMetadata:
     assert type(uploaded_to_pypi) is bool
     partial_stub: object = data.get("partial_stub", True)
     assert type(partial_stub) is bool
+    requires_python_str: object = data.get("requires_python")
+    if requires_python_str is None:
+        requires_python = None
+    else:
+        assert isinstance(requires_python_str, str)
+        requires_python = Specifier(requires_python_str)
+        # Check minimum Python version is not less than the oldest version of Python supported by typeshed
+        assert Specifier(f">={OLDEST_SUPPORTED_PYTHON}").contains(
+            requires_python.version
+        ), f"'requires_python' contains versions lower than the oldest supported Python {OLDEST_SUPPORTED_PYTHON}"
 
```

Comment:
Should we add a check that `requires-python` only specifies the minimum version? Something like
```python
assert requires_python.operator == ">="
```

Reply:
Sounds reasonable to me